### DISCUSSION TITLE
Should we use localisation ?

### DISCUSSION THREAD

**Discussion by pierresisson** [NostrGit/NostrGit#81] [General]
only english or should we use localisation ? 

https://en.wikipedia.org/wiki/Language_localisation

I prefer ask now because we need to implement it as soon as possible if we choose to localize

**Comment by PeerRich** [comment]
lets focus on english for now but later use next-i18next + crowdin.com (with github actions sync), they have a free enterprise tier for open source and are very good

**Comment by pierresisson** [comment]
didnt know, thx

**Comment by cypherhoodlum** [comment]
We [decided](https://github.com/NostrGit/NostrGit/discussions/95) to not use Discussions anymore. Let's continue the discussion at #123  :heart:
